src/testE2E/codecatalyst/client.test.ts

```diff
@@ -132,7 +132,11 @@ describe('Test how this codebase uses the CodeCatalyst API', function () {
         })
 
         describe('getThisDevEnv', function () {
-            const ccAuth = CodeCatalystAuthenticationProvider.fromContext(globals.context)
```

Comment:
Yeah we should never do non-constant setup in the describe() scope because its scope is not deterministically evaluated. Always must be done in before().